doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,29 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms currently supports creating content without an intermediate draft.
+
+Support must be added for editing content. It implies creating a new version of an existing content,
+rendering a form with the previous version's fields values, and allowing the user to save the draft,
+cancel or publish the new version. It should also allow choosing a language, either for translation
+or 
+
+Ideally, the draft would only be created when we need it, not on the first display.
```

Comment:
>  the draft would only be created when we need it, not on the first display.

depends, this can break the UI protection we have against several editing draft.

So imho what we should probably aim for is that API takes care about this, creating internal draft when you can for creating draft, and not change it to real draft before updates on it happens _(on store)_.